composer.json

```diff
@@ -10,7 +10,7 @@
     }
   ],
   "require": {
-    "php": "^7.2",
+    "php": "^7.2|^8.0",
```

Comment:
I'm not totally sure that we support all the approved features of php 8. I think this move is more complicated...